components/scanners/cdxgen/component.yaml

```diff
@@ -0,0 +1,30 @@
+name: cdxgen
```

Comment:
Did you run this with smithyctl and is parsed ok?

Reply:
yes